lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -609,13 +610,18 @@ def _section_send_email(course, access):
         # xblock rendering.
         request_token=uuid.uuid1().get_hex()
     )
+    cohorts = []
+    if is_course_cohorted(course_key):
+        cohorts = [cohort for cohort in get_course_cohorts(course)]
```

Comment:
These three lines look a little verbose. `get_course_cohorts()` already returns a list (so the list comprehension is redundant), and it seems to return an empty list if the course doesn't have any cohorts, so the `is_course_cohorted` check may be unnecessary.  (unless `get_course_cohorts` might return cohorts for a course that is not technically cohorted--the documentation at get_course_cohorts is slightly vague about that).